src/main/java/com/communityratesgames/Application.java

```diff
@@ -9,6 +9,6 @@
     public static void main(String[] args) {
 
         SpringApplication.run(Application.class, args);
-        System.out.println("Hallå eller gubbar!");
+        System.out.println("###Code Compiled Sucessfully###");
     }
 }
```

Comment:
I noticed that some of the files don't end with a newline, as seen here.
Just want to emphasize the importance of it; Git, as well as most other UNIX programs, expects files to end with a newline, and not doing so can confuse these programs. See https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline for more explanation.

Reply:
I've fixed this now, but please think about this in the future.